packages/CopyInput/README.md

```diff
@@ -0,0 +1,58 @@
+<!-- autogenerated don't modify -->
+
+# @paprika/copy-input - 1.0.0
+
+## Description
+
+The CopyInput component is an input with a button that copies the input's contents to user's clipboard.
+
+## Installation
+
+```
+yarn add @paprika/copy-input
+```
+
+or with npm:
+
+```
+npm install @paprika/copy-input
+```
+
+## Props
+
+### CopyInput
+
+| Prop         | Type   | required | default | Description                  |
+| ------------ | ------ | -------- | ------- | ---------------------------- |
+| children     | node   | false    | null    | Used for CopyInput.Input     |
+| defaultValue | string | false    | ""      | Default value for the input. |
+
+<!-- autogenerated don't modify -->
+<!-- content -->
+
+### Usage
+
+```js
+import CopyInput from "@paprika/copy-input";
+
+<CopyInput defaultValue="Copy me" />;
+```
+
+Providing more props to the internal `Input`
+
+```js
+import CopyInput from "@paprika/copy-input";
+
+<CopyInput defaultValue="Copy me">
+  <CopyInput.Input isReadOnly />
+</CopyInput.Input>;
```

Comment:
As a user, I'd love to see a real-world usage example, including controlling the Input.